### ISSUE TITLE
Won't install...At all!

### ISSUE THREAD

**Issue by anonnymouse** [humhub/humhub#565]
I don't know what the issue is, i've tried many times to get humhub to install and it wont work.

I first tried it on my first site which I was using Hostinger to host, I had all the files uploaded and i'm not sure how but I got it to install, but then there were errors and it didn't install properly so I deleted it all and tried again apart from this time I tried everything and nothing brought up the install window I had before-Deciding this was probably due to it being previously installed on there I set up a new domain and again i'm hosting this using Hostinger(I dont know if this is important, Just feel I should include it) but now i'm having a similar issue, I downloaded the zip file, Uploaded and unzipped it, then I cannot get it to install, I believe I have got to the correct path for installing buti'm getting this error:

"PHP warning

mkdir(): Disk quota exceeded

/home/u823944123/public_html/humhub-master/humhub-master/protected/vendors/yii/utils/CFileHelper.php(307)

295      * @param array $options newDirMode element used, must contain access bitmask
296      * @param boolean $recursive whether to create directory structure recursive if parent dirs do not exist
297      * @return boolean result of mkdir
298      * @see mkdir
299      */
300     private static function mkdir($dst,array $options,$recursive)
301     {
302         $prevDir=dirname($dst);
303         if($recursive && !is_dir($dst) && !is_dir($prevDir))
304             self::mkdir(dirname($dst),$options,true);
305 
306         $mode=isset($options['newDirMode']) ? $options['newDirMode'] : 0777;
307         $res=mkdir($dst, $mode);
308         @chmod($dst,$mode);
309         return $res;
310     }
311 }"

However I don't see how it is a disk space issue because i've used practically none:
INODES =0/20000
DISK USAGE =0.02 / 2000 MB
BANDWIDTH =0.00 / 100000 MB

Basically i'm just really really lost.

**Comment by anonnymouse** [comment]
Okay I believe I solved my own issue, I'll leave it up in case anybody has a similar issue, Its something about the ZIP file being too big to unzip in the file thing (Im being very factual, I know) but anyway.
If you go onto your hosting/server thing(If you're using an online one, I dont know about other ones) and go under website-> Import website, then upload it that way, You should be able to upload it without it complaining about the file size and it will unzip it, Then just go to the folder and click the index.php or the file that is really similar to that name.
